src/rickbox/app.py

```diff
@@ -1,12 +1,25 @@
 from flask import Flask
 
+from blueprints.register import register_all_blueprints
+from database import RecordNotFoundError
+from database.models.user import User
+
 app = Flask(__name__)
+register_all_blueprints(app)
```

Comment:
For CLI usage like `flask db init`, `flask user create`, etc.

- Any particular functionality you want implemented?
- Any stuff that's here that you want removed? (lol, flask db drop) I can just move that stuff to an outside shell script in .gitignore -- I added this stuff for my own convenience mostly.